antithesis_sdk.h

```diff
@@ -4,8 +4,28 @@
 //
 // Documentation for the SDK is found at https://antithesis.com/docs/using_antithesis/sdk/cpp/overview/.
 
+#if __cplusplus < 202000L
+    #error "The Antithesis C++ API requires C++20 or higher"
+    #ifndef NO_ANTITHESIS_SDK
+        #define NO_ANTITHESIS_SDK
+    #endif
+#endif
+
+#if defined(__clang__) && __clang_major__ < 16
+    #error "The Antithesis C++ API requires clang version 16 or higher"
+
```

Comment:
Similarly to above; anything below `#error` is useless (it will not be used during compilation) because `#error` will stop the compilation with diagnostic http://www.eel.is/c++draft/cpp.error